### DISCUSSION TITLE
NCSU Workshop - Demos results (November 18th 2022)

### DISCUSSION THREAD

**Discussion by paularamo** [openvinotoolkit/openvino_notebooks#657] [Show and tell]
We want to create a better dynamic with the audience, we will use this thread for sharing results during the tutorial.

**Comment by kunalshah03** [comment]
Successfully ran the notebook.[
![Screenshot (68)](https://user-images.githubusercontent.com/48029057/202788679-84dea644-7aaa-46be-8d6a-69b26b2e5cf7.png)
](url)

**Comment by ghost** [comment]
Ran the code successfully!
I dont have a webcam though :(

![image](https://user-images.githubusercontent.com/111795753/202788788-e4210b7d-1b4c-4a0b-b051-9b7797afda46.png)

**Comment by esogea** [comment]
Successfully ran the notebook on my face and dog.

![401 result](https://user-images.githubusercontent.com/29362344/202789041-6459b52a-03de-4ecb-b9b8-efc9ad85bbbb.png)

![401 result dog](https://user-images.githubusercontent.com/29362344/202789459-467d159e-e8b1-4461-8fa4-a48da5250ed5.png)

**Comment by priyajakhar** [comment]
![image](https://user-images.githubusercontent.com/80418210/202788408-1ae0698b-399f-49aa-9979-adb60fc4756b.png)

**Comment by priyajakhar** [comment]
![image](https://user-images.githubusercontent.com/80418210/202789681-5d3ba09c-2691-40fa-825b-9be83e8d5434.png)

**Comment by tomas19** [comment]
Successfully ran the pose estimation notebook

![image](https://user-images.githubusercontent.com/16766865/202789730-b128b391-da20-4890-82e2-8713b15a45f0.png)

**Reply by tomas19** [reply]
intel i5 7th gen, 6 years old laptop

**Comment by esogea** [comment]
Task #1 Byron Qi

i7 8th gen
![402 result](https://user-images.githubusercontent.com/29362344/202790047-fae30cd9-7894-492f-840d-eb5917b278d3.png)

**Comment by Patjim22** [comment]
![image](https://user-images.githubusercontent.com/33301533/202790199-7ca5f315-a9ab-4c0c-92cb-e295a4009ce3.png)
i7 9th gen

**Comment by MEKNC3** [comment]
![FPS](https://user-images.githubusercontent.com/110567393/202790230-4aa1e42e-3563-4c69-a66f-0819af233981.png)

**Comment by XingjianL** [comment]
![image](https://user-images.githubusercontent.com/43093990/202790330-ffa453c5-a70d-4b1d-9307-20f05ee424fa.png)
dont have access to webcam in my environment.

**Comment by Morsus98** [comment]
![image](https://user-images.githubusercontent.com/69538393/202790450-f714fa54-0380-40d7-8103-fbf314329844.png)
No webcam access. Only around 12 fps.

**Comment by TrentonWallis** [comment]
Not very many frames, but still works on my 5 year old laptop!

![image](https://user-images.githubusercontent.com/65638400/202790516-43c2fb2d-4314-415a-a1eb-bcdac6fd6494.png)

**Comment by cpetrykowski** [comment]
134 fps
![image](https://user-images.githubusercontent.com/100293988/202790530-1bed2a79-6196-4d99-9944-d0345901e104.png)

**Comment by Perseus101** [comment]
Pose estimation running at 45 fps out of the box!

![image](https://user-images.githubusercontent.com/7926482/202790583-05065540-33c2-4cc2-9188-10d48f665989.png)

**Comment by KStickney** [comment]
![image](https://user-images.githubusercontent.com/31007720/202790570-85a4dfe2-4a3e-4141-8a0f-49140693155f.png)

6 CPUS - Old computer

**Comment by ambatimuthya** [comment]
<img width="411" alt="image" src="https://user-images.githubusercontent.com/37538422/202790481-d528827b-f925-4188-80b9-286abf417207.png">

approx 16fps

**Comment by anschoon** [comment]
11.5 FFS
![Capture](https://user-images.githubusercontent.com/58271585/202790614-cf2250ee-6f8f-4e9c-9244-30ad5c6611cb.PNG)

**Comment by Sumukha21** [comment]
![image](https://user-images.githubusercontent.com/45757780/202790616-43089e12-f210-40c5-a88b-c6d287f47dc3.png)

**Comment by rahulgautam21** [comment]
![image](https://user-images.githubusercontent.com/12576405/202790631-364e68c3-cda9-42d1-bebc-40c1d25b101b.png)

**Comment by nathankohen** [comment]
![image](https://user-images.githubusercontent.com/99231385/202790644-9d9057e4-0358-471d-ad7a-afa171fd5e0a.png)

**Comment by Karthikg99** [comment]
Ran Notebook 401
![Screenshot (10)](https://user-images.githubusercontent.com/88997741/202790712-b6e0f40e-55c9-43bc-be5c-10292b0fbca6.png)

**Comment by ghost** [comment]
Task #1 
Neha Kale
Around 25 FPS using CPU

![image](https://user-images.githubusercontent.com/111795753/202790987-8ac171b0-54db-4c77-a434-4cc1b9e2a4a0.png)

**Comment by kunalshah03** [comment]
i7 8th Gen
![Screenshot (69)](https://user-images.githubusercontent.com/48029057/202791189-66602cfa-43c7-48df-a01d-1b0c29f7033d.png)

**Comment by derekchang4** [comment]
![Screenshot_20221118_025117](https://user-images.githubusercontent.com/103058617/202791006-68e1709b-3287-48d6-81f9-592088b561bb.png)
I got a whole 15 FPS
i7, 11th Gen

**Comment by mjmathew7** [comment]
![image](https://user-images.githubusercontent.com/118567159/202791691-665131c1-6a4c-4c46-a5c2-df999c626f37.png)
 18 FPS sheesh

**Comment by Karthikg99** [comment]
8 FPS on notebook 402, not bad for a 6th gen intel processor ig lol
![Screenshot (12)](https://user-images.githubusercontent.com/88997741/202791905-a96302ae-9e56-4a26-ba00-c8c8eed19173.png)

**Comment by anuPh23** [comment]
![image](https://user-images.githubusercontent.com/19801212/202792652-20d3b314-612a-4468-b229-531578c53f9e.png)

**Comment by priyajakhar** [comment]
![image](https://user-images.githubusercontent.com/80418210/202792932-6de48754-3e37-4ef6-a1e0-91467561aff2.png)

**Comment by esogea** [comment]
Task #2 Byron Qi
Interesting results:
![205 result](https://user-images.githubusercontent.com/29362344/202792937-579241fe-9def-46cd-a393-d06720b51f95.png)

Original background image:
![rainbow_background](https://user-images.githubusercontent.com/29362344/202792975-c2d84c48-3d6f-4eeb-a777-b8b7654137e4.png)

**Comment by Sumukha21** [comment]
![image](https://user-images.githubusercontent.com/45757780/202793008-9f562af5-f38a-4174-bebb-fb3269e143c0.png)

**Comment by KStickney** [comment]
![image](https://user-images.githubusercontent.com/31007720/202793018-001a9988-6802-4b37-9c58-b961f1baa073.png)

**Comment by XingjianL** [comment]
![modified_horse](https://user-images.githubusercontent.com/43093990/202793214-aaf057d5-68f4-49e8-9248-54e02b608c91.jpg)
![horse](https://user-images.githubusercontent.com/43093990/202793237-1df3c803-cf4d-4ade-b789-3781bbfac4e7.jpg)
Task #2 Xingjian Li

**Comment by anschoon** [comment]
![asdfasdf](https://user-images.githubusercontent.com/58271585/202793307-083a3afa-26ea-4273-acc1-072016655412.PNG)

**Comment by priyajakhar** [comment]
![image](https://user-images.githubusercontent.com/80418210/202793288-60ad7d9f-00eb-4184-9324-d0ee08cdbd3e.png)
Just me adoring a dog

**Comment by Perseus101** [comment]
Dropped coco into Lake Twenty Two, Washington!

![coco_lake_twenty_two](https://user-images.githubusercontent.com/7926482/202793353-f8e972f8-4f13-45fe-9618-c157be15b6a6.png)

Original Image:
![20220908_150824](https://user-images.githubusercontent.com/7926482/202793227-5629fd3f-fbdc-44e5-914f-7366c81158fb.jpg)

**Reply by rahulgautam21** [reply]
![image](https://user-images.githubusercontent.com/12576405/202793440-f9fde874-e47b-41db-8146-91faa94aafbf.png)

**Comment by Karthikg99** [comment]
Coco travels the world in seconds!!!
![Screenshot (13)](https://user-images.githubusercontent.com/88997741/202793397-29a3eeae-015b-4309-93fe-12d91fa8f7cc.png)

**Comment by anuPh23** [comment]
![image](https://user-images.githubusercontent.com/19801212/202793490-f93ae533-c088-41ef-94ff-b3fd4b6cf13b.png)

**Comment by kunalshah03** [comment]
![Screenshot (70)](https://user-images.githubusercontent.com/48029057/202793549-6c83126b-1de7-474d-bf88-61b2ef742f42.png)

**Comment by tomas19** [comment]
![image](https://user-images.githubusercontent.com/16766865/202793575-f707d5a7-6bcd-42cc-a07c-49fd863932f7.png)

**Comment by MEKNC3** [comment]
![Cursed](https://user-images.githubusercontent.com/110567393/202793565-fec401b5-b92f-4bae-8d6f-57f74c3ac205.png)

**Comment by ambatimuthya** [comment]
<img width="710" alt="image" src="https://user-images.githubusercontent.com/37538422/202793611-98c07dbb-c69a-4a89-997a-0fc138fd659d.png">

**Comment by Patjim22** [comment]
![image](https://user-images.githubusercontent.com/33301533/202793613-0e1556fb-e9e6-47d7-9ed7-c31d497161ca.png)

**Reply by raymondlo84** [reply]
LOL!!!

**Comment by nathankohen** [comment]
![image](https://user-images.githubusercontent.com/99231385/202793704-b991dbd5-0fbe-4f08-9c4f-2191d6c2f541.png)

**Comment by anuPh23** [comment]
![image](https://user-images.githubusercontent.com/19801212/202793723-5a246d20-91e1-46e9-9284-932ca41c6925.png)

**Comment by sdutta2000** [comment]
Task 2
![Screenshot (3)](https://user-images.githubusercontent.com/118567328/202793763-48dfa5cc-d696-40d4-a98d-869065d8f560.png)

**Comment by derekchang4** [comment]
![Screenshot_20221118_031239](https://user-images.githubusercontent.com/103058617/202794263-4974422c-c257-421c-9dd3-0eea3e6c57db.png)

**Comment by Aryabb29** [comment]
<img width="781" alt="image" src="https://user-images.githubusercontent.com/48296440/202794656-53257b73-3002-41b5-835c-2a836671b137.png">

**Comment by ghost** [comment]
Task # 2


![image](https://user-images.githubusercontent.com/111795753/202795111-ced8a40c-9abf-405e-986a-ddd5d970d82d.png)

**Comment by Hbanah** [comment]
<img width="960" alt="2022-11-18" src="https://user-images.githubusercontent.com/88856587/202796659-a0a7a138-ddec-453b-abbb-dc5a48dc7de1.png">
